drivers/virtualbox/vbm.go

```diff
@@ -56,6 +56,19 @@ func setVBoxManageCmd() string {
 	return cmd
 }
 
+// VBoxManager defines the interface to communicate to VirtualBox.
+type VBoxManager interface {
+	queryVirtualBox(args ...string) (string, string, error)
+}
+
+// VBoxCmdManager will communicate with VirtualBox through the commandline
+// using `VBoxManage`.
+type VBoxCmdManager struct{}
+
+func (v *VBoxCmdManager) queryVirtualBox(args ...string) (string, string, error) {
+	return vbmOutErr(args...)
```

Comment:
Looks good.  Longer-termish, we should think about embedding all related logic (e.g. finding the path of `VBoxManage`) into `VBoxCmdManager` directly, but this is an acceptable solution to get the ball rolling on testing.